experiment/config-forker/main.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+	"log"
+	"strings"
+
+	"k8s.io/test-infra/prow/config"
+	"sigs.k8s.io/yaml"
+)
+
+type options struct {
+	jobConfig  string
+	outputPath string
+	newVersion string
+}
+
+func parseFlags() options {
+	o := options{}
+	flag.StringVar(&o.jobConfig, "job-config", "config/", "Path to the job config")
+	flag.StringVar(&o.outputPath, "output", "", "Path to the output yaml")
+	flag.StringVar(&o.newVersion, "version", "", "Version number to generate jobs for")
```

Comment:
blow up if this isn't a good value?

Reply:
will clean up this whole file after I refactor for postsubmits.